src/sentry/models/integrations/integration.py

```diff
@@ -62,9 +90,15 @@ class Meta:
         unique_together = (("provider", "external_id"),)
 
     def get_provider(self) -> IntegrationProvider:
-        from sentry import integrations
+        return get_integration_provider(integration=self)
+
+    def get_installation(self, organization_id: int, **kwargs: Any) -> IntegrationInstallation:
+        return get_integration_installation(
+            integration=self, organization_id=organization_id, **kwargs
+        )
```

Comment:
I like this approach to providing shared interfaces between ORM and RPC models. Nice work :clap: